### DISCUSSION TITLE
Requesting touch controls for Halo: The Master Chief Collection

### DISCUSSION THREAD

**Discussion by BeebisToo** [redphx/better-xcloud#380] [Custom touch controls]
I would like to request custom touch controls for Halo MCC https://www.xbox.com/en-CA/play/launch/halo-the-master-chief-collection/9MT8PTGVHX2P

I have made a rough mockup of what touch controls could look like. This is based off of the Universal Default (Recon) controller layout.
![HALO TOUCH CONTROL MOCKUP](https://github.com/redphx/better-xcloud/assets/168618895/bfc150f2-a4f6-402e-bc0c-b3cd553765bc)
Graphic design is NOT my passion I apologize for it looking cluttered.

**Comment by redphx** [comment]
xCloud's touch control doesn't support this type of layout
